robyn/__init__.py

```diff
@@ -395,12 +395,23 @@ def inner(handler):
 
         return inner
 
-    def include_router(self, router):
+    def include_router(self, router, added_routes=None):
         """
         The method to include the routes from another router
 
         :param router Robyn: the router object to include the routes from
```

Comment:
why do we need an `added_route` param here? Could you please the usage in the docstring?

Reply:
Please also add the typing here.

Reply:
I'm sorry I want to comment in this conversation but reply on another conversation instead. My mistake.
https://github.com/sparckles/Robyn/pull/889#discussion_r1676771826